Base.rte/Scenes/Objects/Bunkers/BunkerSystems/BunkerSystems.ini

```diff
@@ -75,4 +75,136 @@ AddTerrainObject = TerrainObject
 			Y = -39
 		HFlipped = 1
 		Rotation = Matrix
-			AngleDegrees = 90
\ No newline at end of file
+			AngleDegrees = 90
+
+// This particle is for creating cracks around the destroyed node
+AddParticle = MOPixel
+	PresetName = Node Spark
+	Mass = 1
+	RestThreshold = 500
+	LifeTime = 10
+	Sharpness = 10
+	HitsMOs = 0
+	GetsHitByMOs = 0
+	Color = Color
+		R = 255
+		G = 255
+		B = 255
+	Atom = Atom
+		Material = Material
+			CopyOf = Air Blast
+		TrailColor = Color
+			R = 255
+			G = 255
+			B = 255
+		TrailLength = 25
+
+
+AddActor = MOSRotating
+	PresetName = Powered Ladder Node
+	PinStrength = 1000
+	Mass = 20
+	RestThreshold = 0
+	HitsMOs = 0
+	GetsHitByMOs = 1
+	SpriteFile = ContentFile
+		FilePath = Base.rte/Scenes/Objects/Bunkers/BunkerSystems/LadderNode.bmp
+	FrameCount = 1
+	SpriteOffset = Vector
+		X = -12
+		Y = -12
+	ScriptPath = Base.rte/Scenes/Objects/Bunkers/BunkerSystems/LadderNode.lua
+	EntryWound = AEmitter
+		CopyOf = Dent Metal
+	ExitWound = AEmitter
+		CopyOf = Dent Metal
+	AtomGroup = AtomGroup
+		AutoGenerate = 1
+		Material = Material
+			CopyOf = Military Stuff
+		Resolution = 1
+		Depth = 0
+	DeepGroup = AtomGroup
+		AutoGenerate = 1
+		Material = Material
+			CopyOf = Military Stuff
+		Resolution = 4
+		Depth = 6
+	DeepCheck = 0
+	AddGib = Gib
+		GibParticle = MOSParticle
+			CopyOf = Gib Glass Micro A
+		Count = 3
+	AddGib = Gib
+		GibParticle = MOPixel
+			CopyOf = Spark Yellow 1
+		Count = 6
+		Spread = 3.14
+		MaxVelocity = 30
+		MinVelocity = 15
+		LifeVariation = 0.30
+	AddGib = Gib
+		GibParticle = MOSParticle
+			CopyOf = Tiny Smoke Ball 1
+		Count = 3
+		Spread = 3.14
+		MaxVelocity = 30
+		MinVelocity = 15
+		LifeVariation = 0.30
+	AddGib = Gib
+		GibParticle = MOPixel
+			CopyOf = Node Spark
+		Count = 9
+		Spread = 3.14
+		MaxVelocity = 30
+		MinVelocity = 15
+	GibImpulseLimit = 500
+	GibWoundLimit = 5
+	GibSound = Sound
+		CopyOf = Robot Death
+
+
+AddTerrainObject = TerrainObject
+	PresetName = Powered Ladder Left
```

Comment:
Put all the ladder stuff into a dedicated folder.
Generally if something has more than 2 sprites and 3 lines of code it should be in a dedicated folder.